skietbaan_fe/src/actions/postActions.js

```diff
@@ -1,31 +1,76 @@
+import { PASS_ID,FETCH_POSTS,UPDATE_GROUPNAME,UPDATENAME,CREATEGROUP,GETGROUP,GETNAME,URLADD,URLUSER,URLGROUP,FETCH_LEADERBOARDFILTER_DATA ,FETCH_LEADERBOARDTABLE_DATA} from './types';
```

Comment:
Refer to my comment I made towards the end. Once you have don't that you will not have to import all of these individually. Also, this line is long so you need to split it over multiple lines. It is standard one import per line so it will be like 
```
import {
 PASS_ID,
 FETCH_POSTS,
 UPDATE_GROUPNAME
}
```